client/src/components/SingleView/Messages.jsx

```diff
@@ -0,0 +1,234 @@
+import React, { useContext, useEffect, useState } from "react";
+import axios from "axios";
+import UserContext from "context/UserContext";
+import {
+  Typography,
+  Avatar,
+  Button,
+  TextField,
+  Portal,
+  Snackbar,
+} from "@material-ui/core";
+import EditIcon from "@material-ui/icons/Edit";
+import CloseIcon from "@material-ui/icons/Close";
+import { Alert } from "@material-ui/lab";
+import CodeEditor from "components/CodeEditor";
+import formatDate from "utils/formatDate";
+import { getToken } from "utils/storage";
+import useStyle from "components/SingleView/SingleView.css";
+
+const Messages = ({ message, language, requestId }) => {
+  const classes = useStyle();
+  const { user } = useContext(UserContext);
+  const [editMode, setEditMode] = useState(false);
+  const [errorSnackbar, setErrorSnackbar] = useState({
+    open: false,
+    message: "",
+  });
+  const [successSnackbar, setSuccessSnackbar] = useState({
+    open: false,
+    message: "",
+  });
```

Comment:
When I did something like this before, Shums had me combine these into a single snackbar. You may want to take a look at pages/Balance/Checkout.jsx. The idea is to add an extra property to the object called `severity` ("" initial value, "error" or "success", and onClose you don't need to change the value of `severity`).

Reply:
Thanks, Updated!